### DISCUSSION TITLE
Regex for multiline RemNote style

### DISCUSSION THREAD

**Discussion by dummifiedme** [ObsidianToAnki/Obsidian_to_Anki#212] [Q&A]
I am trying to have a multi-line RemNote style. I had seen it somewhere in the Issues but I don't seem to find it. If someone can help me out, please.

The card was supposed to be like this:

```
This is a question ::: 
This is Answer
This too
```

also, I hoped it works for bulleted lists:

```
- This is a bullet question ::: 
    - This is Answer
    - This too
```

**Comment by insecurejezza** [comment]
This can be achieved using the paragragh cloze.

This is a question:
{This is Answer}
{This too}
